### PR TITLE
Add Ansible-Galaxy release workflow

### PR DESCRIPTION
**Pull request by sanjaysrikakulam**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      - name: Set up Python 3.
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+
+      - name: Install Ansible.
+        run: pip3 install ansible-core
+
+      - name: Trigger a new import on Galaxy.
+        run: ansible-galaxy role import --api-key ${{ secrets.ANSIBLE_GALAXY_API_KEY }} $(echo ${{ github.repository }} | cut -d/ -f1) $(echo ${{ github.repository }} | cut -d/ -f2) --branch master
```
Comment (mira-miracoli) [inline .github/workflows/release.yml:32]: maybe we can also cut off the "ansible", [currently](https://galaxy.ansible.com/usegalaxy_eu/gie_proxy) the role is named "gie_proxy"
We should change that in other roles, too

**Comment by sanjaysrikakulam** [comment]
I think the release through the CI will create a new `gie-proxy` role instead of pushing it to the existing 'gie_proxy' [role](https://galaxy.ansible.com/usegalaxy_eu/gie_proxy). So, please review this carefully as we do not want to create additional roles. If in case this is going to create a new role, please reject the PR and whoever has access to the _usegalaxy_eu's_ Ansible-Galaxy please push the v0.0.3 to the 'gie_proxy' [role](https://galaxy.ansible.com/usegalaxy_eu/gie_proxy)

File: .github/workflows/release.yml (hunk 1 above)
Comment (mira-miracoli) [reply .github/workflows/release.yml:32]:
```diff
-         run: ansible-galaxy role import --api-key ${{ secrets.ANSIBLE_GALAXY_API_KEY }} $(echo ${{ ithub.repository }} | cut -d/ -f1) $(echo ${{ github.repository }} | cut -d/ -f2) --branch master
+         run: ansible-galaxy role import --api-key ${{ secrets.ANSIBLE_GALAXY_API_KEY }} $(echo ${{ ithub.repository }} | cut -d/ -f1) $(echo ${{ github.repository }} | cut -d/ -f2) --role-name gie_proxy --branch master
```
I think this should do the job

**Comment by bgruening** [comment]
Should we not create a new one, with the proper name - called `gx-it-proxy`?

**Comment by bgruening** [comment]
ping @natefoo @hexylena

**Comment by sanjaysrikakulam** [comment]
> Should we not create a new one, with the proper name - called `gx-it-proxy`?

We have a git repo called [gx-it-proxy](https://github.com/galaxyproject/gx-it-proxy) that's cloned by this ansible role to set up the proxy. How about `gxit_proxy` instead of the existing `gie_proxy` for this role?

**Comment by hexylena** [comment]
I mean from our side we don't want this at all, it'll be part of the main role and gravity's job to install and manage, making some of these older individual roles (also tusd) less useful.

**Comment by hexylena** [comment]
re the name, I don't think that's what will happen. the name is taken from the meta.

**Comment by sanjaysrikakulam** [comment]
> re the name, I don't think that's what will happen. the name is taken from the meta.

Oh, yeah! Thank you!

**Comment by bgruening** [comment]
> I mean from our side we don't want this at all, it'll be part of the main role and gravity's job to install and manage, making some of these older individual roles (also tusd) less useful.

This assumes you have tusd on the headnode, or need gravity on an external node to manage tusd?

**Comment by hexylena** [comment]
Yes. (Though I suspect, @natefoo can confirm) it will look a lot like

headnode: run ansible-galaxy, install galaxy
uploads: run ansible-galaxy, no install galaxy, install tusd

**Comment by sanjaysrikakulam** [comment]
ping @bgruening, @mira-miracoli

**Review by mira-miracoli — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (hexylena) [reply .github/workflows/release.yml:32]: the role name is specified in meta/meta.yaml. I would not put it here, that would prevent reusing this YAML as is (like we do across other galaxy roles)

**Comment by natefoo** [comment]
Eventually this will be folded into [the collection](https://github.com/galaxyproject/ansible-collection-galaxy) but it's fine continuing as its own thing for now. I don't want to add anything else to the monolithic ansible-galaxy role.

**Pull request merged**